instagram/src/components/CommentSection.js

```diff
@@ -1,21 +1,79 @@
 import React from 'react';
 import Comment from './Comment'
+import moment from 'moment'
+import Likes from './Likes'
 
+class CommentSection extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        commentList: props.commentList,
+        commentText: "", 
+        likes: props.obj.likes
```

Comment:
If you're not going to use your likes here you shouldn't set them to state and then pass them down. You already have your likes in your props. Just pass down the likes again from your props down to your Likes component.